packages/http/src/mocker/generator/HttpParamGenerator.ts

```diff
@@ -51,7 +52,7 @@ function pickStaticExample(
   );
 }
 
-export function generate(param: IHttpParam): Option.Option<unknown> {
+export function generate(param: { schema?: JSONSchema4 | JSONSchema6 | JSONSchema7; examples?: Array<INodeExample | INodeExternalExample> }): Option.Option<unknown> {
```

Comment:
What is this change about?

Reply:
I hit a problem that `IHttpParam` (https://github.com/stoplightio/types/blob/master/src/http-spec.ts#L72) and `IHttpContent` (https://github.com/stoplightio/types/blob/master/src/http-spec.ts#L123) are separate interfaces. So I changed `generate()` signature to define only the fields it needs to operate (`examples` and `schema`). Side effect is that we can put `IHttpParam` and `IHttpContent` because the field names match :)